docs/development/plugin-creation.md

```diff
@@ -1,37 +1,37 @@
-# Plugin Creation
+# Plugin creation
 
-Plugins can simply be implementations of [triggers](/docs/development/plugin-triggers.md) or they may implement a command structure of their own. Dokku has no restrictions on the language in which a plugin is implemented, it only cares that the plugin implements the appropriate [commands](/docs/development/plugin-creation.md#command-api) or [triggers](/docs/development/plugin-triggers.md) API. NOTE: all files that implement the triggers or commands API must be executable.
+A plugin can be a simple implementation of [triggers](/docs/development/plugin-triggers.md) or can implement a command structure of its own. Dokku has no restrictions on the language in which a plugin is implemented; it only cares that the plugin implements the appropriate [commands](/docs/development/plugin-creation.md#command-api) or [triggers](/docs/development/plugin-triggers.md) for the API. **NOTE:** any file that implements triggers or uses the command API must be executable.
 
 If you create your own plugin:
 
-1. Take a look at the plugins shipped with Dokku and hack away!
-2. Check out the [list of triggers](/docs/development/plugin-triggers.md) your plugin can implement.
-3. Upload your plugin to github with a repository name in form of `dokku-<name>` (e.g. `dokku-mariadb`)
-4. Edit [this page](/docs/community/plugins.md) and add a link to it.
+1. Take a look at [the plugins shipped with Dokku](https://github.com/dokku/dokku/blob/master/docs/community/plugins.md) and hack away!
```

Comment:
Can you change the url back to `/docs/community/plugins.md`? That will make it work both on GitHub and on our docs site.